### PR TITLE
Use cita-logger crate and 2018 edition.

### PR DESCRIPTION
**Pull request by kaikai1024**

### CODE REVIEW

**Review by keroro520 — APPROVED** [review]

File: src/main.rs [hunk 1]
```diff
@@ -15,10 +15,9 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-extern crate clap;
 extern crate crossbeam_channel as channel;
 #[macro_use]
-extern crate logger;
+extern crate cita_logger;
```
Comment (leeyr338) [inline src/main.rs:20]: Is it no need to `extern` crate in 2018 edition?

Comment (kaikai1024) [reply src/main.rs:20]: Not include `proc_macro`.

https://doc.rust-lang.org/edition-guide/rust-2018/module-system/path-clarity.html#an-exception

Comment (leeyr338) [reply src/main.rs:20]: Okay! 
But I have wrote some code likes:
```rust
// declares a macro
use util::micro_service_init;

// use the macro
micro_service_init!("cita-network", "CITA:network", stdout);
```

before.

Comment (kaikai1024) [reply src/main.rs:20]: I'm not sure. You can have a try and fix it.

**Pull request merged**